AppCUI/src/Controls/NumericSelector.cpp

```diff
@@ -0,0 +1,176 @@
+#pragma once
+
+#include "AppCUI.hpp"
+#include "ControlContext.hpp"
+
+namespace AppCUI::Controls
+{
+using namespace AppCUI::Input;
```

Comment:
astea nu le poți pune în afara namespace-ului? Oricum ești în cpp și nu leak-uiesti nimic în namespace-ul comun

Reply:
fieldul cu valoare editabila cred ca e ok sa fie un feature al numeric selector pe viitor
dat fiind ca are un mouse wheel event pe care il proceseaza nu cred ca e ceva blocant momentan
https://github.com/gdt050579/AppCUI/projects/1#card-67686119

Reply:
si da, ok, le pun in afara namespaceului
e doar preferinta, cum ai zis si tu, nu cred ca are vreun impact direct atata timp cat e in cpp